src/simulator.zig

```diff
@@ -331,6 +331,7 @@ pub const Simulator = struct {
         request_idle_off_probability: u8, // percent
     };
 
+    allocator: std.mem.Allocator,
```

Comment:
Let's either pass this to `core_missing_blocks` or remove `alocator` argument from deinit.